packages/test/snapshots/README.md

```diff
@@ -1,6 +1,103 @@
 # @fluid-internal/test-snapshots
 
+## Syncing snapshot tests content
 If it is the first time you are syncing the content for the snapshot tests, you need to run the following command:
-    `git submodule update --init --recursive`
 
-This will pull all the content down to enable running the snapshot tests.
+```
+git submodule update --init --recursive
+```
+
+This will pull all the content down to enable running the snapshot tests. The content will be under the `content` subdirectory.
+
+You can run the tests by running the following command:
+
+```
+npm run test
+```
+
+## Submit changes to test content
+For submitting changes to the snapshot test content, follow these steps:
+- Submitting the changes to the test content:
+  - Sync the content for the snapshot tests as mentioned [here](#Syncing-snapshot-tests-content).
+  - Go to the `content` subdirectory, create a new branch and update the snapshot content. Ideally, you should not update the content manually but follow one of these:
+    - [Adding new snapshots](#Adding-new-snapshots) - If you are adding new snapshots.
+    - [Updating existing snapshots](#Updating-existing-snapshots) - If you updating existing snapshots.
+  - Create a commit and push the new branch.
+  - Create a PR against `main` branch in the [FluidFrameworkTestData](https://github.com/microsoft/FluidFrameworkTestData) repo. Get the changes approved and merge the PR.
+  - Checkout `main` branch and run `git pull` to ensure it has your new changes.
+- Updating FluidFramework repo to point to the new content:
+  - Go to the FluidFramework repo root. `git status` should show changes under `packages/test/snapshots/content`. For example:
+  ```
+  D:\FluidFramework>git status
+  On branch udpateTestSnasphots
+  Your branch is up to date with 'origin/udpateTestSnasphots'.
+
+  Changes not staged for commit:
+    (use "git add <file>..." to update what will be committed)
+    (use "git restore <file>..." to discard changes in working directory)
+          modified:   packages/test/snapshots/content (new commits)
+
+  no changes added to commit (use "git add" and/or "git commit -a")
+
+  D:\FluidFramework>
+  ```
+  - Commit these changes to a new branch in the [FluidFramework](https://github.com/microsoft/FluidFramework) repo.
+  - Create a PR against `main`, get the changes approved and merge the PR. FluidFramework repo will now point to the new test contents.
+
+### Adding new snapshots
```

Comment:
Just to clarify the semantics here. `new` means `new to the repo` right? Not necessarily `new` as in `new formats` or `recent snapshots` with regards to the framework version that produced them, right?

Reply:
yes, you are right.